src/cr/cube/stripe/assembler.py

```diff
@@ -55,6 +55,24 @@ def means(self):
         """
         return self._assemble_vector(self._measures.means.blocks)
 
+    def pairwise_significance_means_p_vals(self, column_idx):
+        """1D optional np.float64 ndarray of mean difference p_vals for column idx.
```

Comment:
Why would we ever want this in 1D? I don't really get this part.

Reply:
Yes u right this is not needed ... I'll delete it. Sorry my mistake